src/main/steps/common/task-list/components/notification-banner/definitions.ts

```diff
@@ -25,6 +25,7 @@ export enum NotificationType {
   DA_RESPONDENT_BANNER = 'daRespondentBanner',
   CAFFCASS = 'cafcass',
   ORDER_PERSONAL_SERVICE = 'orderPersonalService',
+  DN1 = 'dn1',
```

Comment:
change it  to daApplicationServedByCourtAdminCourtBayliff

Reply:
Changed to daApplicationServedByCourtAdminCourtBayliff